### PR TITLE
[NUI] separate layouting logic of the independent children

### PR DESCRIPTION
**Pull request by dasg34**
This patch introduces `IterateLayoutChildren()` method that return
an enumerable collection of the child layouts that affected by parent
layout.

Independent children will be measured in `OnMeasureIndependentChildren()`.

By using `IterateLayoutChildren()`, we can remove the codes that check
whether child layout exclude from layout flow in `OnMeasure`. and api
users don't need to check `View.ExcludeLayouting` property in custom layouts.

### API Changes ###

 - ACR:

### CODE REVIEW

**Comment by Seoyeon2Kim** [comment]
I think this PR really needs to get Seungho's review because the inheritance of `ScrollableBase` would be changed. (`LayoutGroup` -> `AbsoluteLayout`)
@bshsqa  Could you check this PR please?

**Comment by bshsqa** [comment]
I think this patch looks good to go.
But, currently, we have some issue for scrollableBase and excludeLayouting.
So, If you OK, could we merge this patch after I fix the issue?
Because it can be difficult to find the way to reproduce the issue after the code is changed.

**Review by bshsqa — APPROVED** [review]
This patch looks good to go.
If well tested, this is good to be merged.

File: src/Tizen.NUI/src/public/Layouting/AbsoluteLayout.cs [hunk 1]
```diff
@@ -99,21 +90,17 @@ protected override void OnLayout(bool changed, LayoutLength left, LayoutLength t
...
+                Position2D childPosition = childLayout.Owner.Position2D;
 
-                    LayoutLength childLeft = new LayoutLength(childPosition.X);
-                    LayoutLength childTop = new LayoutLength(childPosition.Y);
+                LayoutLength childLeft = new LayoutLength(childPosition.X);
+                LayoutLength childTop = new LayoutLength(childPosition.Y);
 
-                    childLayout.Layout(childLeft, childTop, childLeft + childWidth, childTop + childHeight, true);
-                }
+                childLayout.Layout(childLeft, childTop, childLeft + childWidth, childTop + childHeight);
```
Comment (bshsqa) [inline src/Tizen.NUI/src/public/Layouting/AbsoluteLayout.cs:103]: We suppose that every child of AbsoluteLayout is independent. So, this needs true at the end of the Layout method.

**Comment by bshsqa** [comment]
Thanks. I think it is good to be merged

**Review by bshsqa — APPROVED** [review]
Thanks. I think it is good to be merged

**Pull request merged**